src/main/java/com/softserve/academy/spaced/repetition/controller/CardImageController.java

```diff
@@ -15,7 +15,7 @@
     private CardImageService cardImageService;
 
     @ResponseStatus(HttpStatus.OK)
-    @DeleteMapping(value = "/api/private/cardImage/{id}")
```

Comment:
in other controllers we have `/` before `api`

Reply:
if / is not necessary on start of link, why we must put them?